docs/src/estimators_cheatsheet.md

```diff
@@ -0,0 +1,298 @@
+
+```math
+ATE_{t_{case}, t_{control}}(\mathbb{P}) = CM_{t_{case}}(\mathbb{P}) - CM_{t_{control}}(\mathbb{P}) 
+```
+
+And it's associated gradient is:
+
+```math
+\begin{aligned}
+\phi_{ATE}(W, T, Y) &= \phi_{CM_{t_{case}}}(W, T, Y) - \phi_{CM_{t_{control}}}(W, T, Y) \\
+&=  H(W, T)(Y − Q_Y(W, T)) + C(W) − ATE_{t_{case}, t_{control}}(\mathbb{P})
+\end{aligned}
+```
+
+with:
+
+```math
+\begin{aligned}
+  \begin{cases}
+    H(W, T) &= \frac{(-\mathbb{1}(T \in (t_{case}, t_{control})))^{T=t_{control}}}{g(W, T)} \\
+    C(W) &= (Q_Y(W, t_{case}) - Q_Y(W, t_{control}))
+  \end{cases}
+\end{aligned}
+```
+
+``H`` is also known as the clever covariate in the Targeted Learning literature (see later).
+
+#### Average Interaction Effect
+
+For simplicity, we only consider two treatments ``T = (T_1, T_2)`` such that: ``T_1: t_{1,control} \rightarrow t_{1,case}`` and ``T_2: t_{2,control} \rightarrow t_{2,case}``. The Average Interaction Effect (AIE) of ``(T_1, T_2)`` is defined as:
+
+```math
+AIE(\mathbb{P}) = CM_{t_{1, case}, t_{2, case}}(\mathbb{P}) - CM_{t_{1, case}, t_{2, control}}(\mathbb{P}) - CM_{t_{1, control}, t_{2, case}}(\mathbb{P}) + CM_{t_{1, control}, t_{2, control}}(\mathbb{P})
+```
+
+And its gradient is given by:
+
+```math
+\begin{aligned}
+\phi_{AIE, }(W, T, Y) &= \phi_{CM_{t_{1,case}, t_{2,case}}}(W, T, Y) - \phi_{CM_{t_{1,case}, t_{2,control}}}(W, T, Y) - \phi_{CM_{t_{1,control}, t_{2,case}}}(W, T, Y) + \phi_{CM_{t_{1,control}, t_{2,control}}}(W, T, Y) \\
+&=  H(W, T)(Y − Q_Y(W, T)) + C(W) − ATE_{t_{case}, t_{control}}(\mathbb{P})
+\end{aligned}
+```
+
+with:
+
+```math
+\begin{aligned}
+  \begin{cases}
+    H(W, T) &= \frac{(-\mathbb{1}(T_1 \in (t_{1, case}, t_{1, control})))^{T_1=t_{1, control}}(-\mathbb{1}(T_2 \in (t_{2, case}, t_{2, control})))^{T_2=t_{2, control}}}{g(W, T)} \\
+    C(W) &= (Q_Y(W, t_{1, case}, t_{2, case}) - Q_Y(W, t_{1, case}, t_{2, control}) - Q_Y(W, t_{1, control}, t_{2, case}) + Q_Y(W, t_{1, control}, t_{2, control}))
+  \end{cases}
+\end{aligned}
+```
+
+For higher-order interactions the two factors ``H`` and ``C`` can be similarly inferred.
+
+## Asymptotic Analysis of Plugin Estimators
+
+The theory is based on the von Mises expansion (functional equivalent of Taylor's expansion) which reduces due to the fact that the gradient satisfies: ``\mathbb{E}_{\mathbb{P}}[\phi_{\mathbb{P}}(Z)] = 0``.
+
+```math
+\begin{aligned}
+\Psi(\hat{\mathbb{P}}) − \Psi(\mathbb{P}) &= \int \phi_{\hat{\mathbb{P}}}(z) \mathrm{d}(\hat{\mathbb{P}} − \mathbb{P})(z) + R_2(\hat{\mathbb{P}}, \mathbb{P}) \\
+&= - \int \phi_{\hat{\mathbb{P}}}(z) \mathrm{d}\mathbb{P}(z) + R_2(\hat{\mathbb{P}}, \mathbb{P})
+\end{aligned}
+```
+
+This suggests that a plugin estimator, one that simply evaluates ``\Psi`` at an estimator ``\hat{\mathbb{P}}`` of ``\mathbb{P}``, is biased. This bias can be elegantly decomposed in four terms by reworking the previous expression:
```

Comment:
It would be good to remark that the R_2 term is defined by this equation and known as the exact second-order remainder. The bias referred to here is "first-order bias" which would also be good to remark.